### DISCUSSION TITLE
Changed email address but didn't have email support setup in docker compose. Help!

### DISCUSSION THREAD

**Discussion by zettawatt** [firefly-iii/firefly-iii#9208] [Firefly III]
I did something really stupid and now I'm trying to figure out how to recover from this. I'm changing my email address in all my accounts and did this in my Firefly III instance without thinking. Now firefly tells me "Until you verify your email address, you cannot login.". Since I didn't have the email stuff setup in my docker compose file, I guess the email got sent to never never land and I'm not sure how to unlock firefly. I would much prefer to not have to enable email just for this operation. Is there an entry I can make or something in the MySQL database to make this go away? Or alternatively, is there a way to retrigger that email being sent and dump the output to the log file so I can get to whatever link it generates that way?

**Comment by JC5** [comment]
You're not the first one ;)

https://docs.firefly-iii.org/references/faq/firefly-iii/using/#i-changed-my-email-address-and-now-i-cant-log-in

**Comment by zettawatt** [comment]
Looks like I should have RTFM :man_facepalming: I'm back up and running. Thank you so much!
